docs/sources/setup-grafana/configure-security/configure-authentication/ldap-ui/_index.md

```diff
@@ -0,0 +1,122 @@
+---
+aliases:
+  - ../../../auth/enhanced-ldap/
+description: Learn about configuring LDAP authentication in Grafana using the Grafana UI.
+labels:
+  products:
+    - cloud
+    - enterprise
+    - oss
+menuTitle: LDAP user interface
+title: Configure LDAP authentication using the Grafana user interface
+weight: 300
+---
+
+# Configure LDAP authentication using the Grafana user interface
+
+This page explains how to configure LDAP authentication in Grafana using the Grafana user interface. For more detailed information about configuring LDAP authentication using the configuration file, refer to [LDAP authentication]({{< relref "../ldap" >}}).
+
+Benefits of using the Grafana user interface to configure LDAP authentication include:
+
+- There is no need to edit the configuration file manually.
+- Quickly test the connection to the LDAP server.
+- There is no need to restart Grafana after making changes.
+
+{{% admonition type="note" %}}
+Any configuration changes made through the Grafana user interface (UI) will take precedence over settings specified in the Grafana configuration file or through environment variables. If you modify any configuration settings in the UI, they will override any corresponding settings set via environment variables or defined in the configuration file.
+
+For more information on how Grafana determines the order of precedence for its settings, please refer to the [Settings update at runtime]({{< relref "../../../configure-grafana/settings-updates-at-runtime" >}}).
+{{% /admonition %}}
```

Comment:
You are referring to a functionality that's deprecated. Maybe it's not a good idea to include it. wdyt?

Reply:
Good catch. Removed as suggested.